bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -459,10 +489,9 @@ async def unwatch(self, user_id: int, reason: str) -> bool:
 
         nomination = active_nomination[0]
         await self.bot.api_client.patch(
-            f"{self.api_endpoint}/{nomination['id']}",
+            f"bot/nominations/{nomination['id']}",
             json={'end_reason': reason, 'active': False}
         )
-        self._remove_user(user_id)
```

Comment:
We'd probably want to pop the user from the cache here, I guess, although in practice it probably wouldn't affect much.

Reply:
Yea agreed, pushed to https://github.com/python-discord/bot/pull/1487/commits/a06367e1691d8ef39e2cf8f8e0d15e5922d29d01